### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Deepak-du**
update some new features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# PDF.js [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js)
-
+# [PDF.js](https://mozilla.github.io/pdf.js/) &middot; [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/mozilla/pdf.js/blob/master/LICENSE) [![npm version](https://img.shields.io/npm/v/pdfjs-dist)](https://www.npmjs.com/package/pdfjs-dist) [![npm-downloads](https://img.shields.io/npm/dm/pdfjs-dist)](https://npmcharts.com/compare/pdfjs-dist?minimal=true) [![Follow Twitter](https://img.shields.io/twitter/follow/pdfjs?style=social)](https://twitter.com/pdfjs)
```
Comment (Snuffleupagus) [inline README.md:1]: It's not really clear that we need to list all of these things at the top of the README, note how e.g. the license information is already listed at the *top* of https://github.com/mozilla/pdf.js

Furthermore, given how infrequent the PDF.js Twitter-account is updated (the last post was 4.5 years ago), it probably doesn't make sense to list that at the top of the README either!?

Hence I'd suggest simplifying this to only:
```suggestion
# [PDF.js](https://mozilla.github.io/pdf.js/) [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js) [![npm version](https://img.shields.io/npm/v/pdfjs-dist)](https://www.npmjs.com/package/pdfjs-dist)
```

**Comment by Deepak-du** [comment]
@timvandermeij please review this

File: README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-# PDF.js [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js)
-
```
Comment (Snuffleupagus) [inline README.md:2]: Please don't remove blank lines, since they aid with readability of the "raw"-file.

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]
Please remember to write a good *commit message*, explaining what you're changing and why, since "Update README.md" isn't terribly descriptive.

Also, please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

**Comment by Snuffleupagus** [comment]
So far it seems that only *some* of the review comments have been addressed here, hence please fix *all* of them and remember both of the points mentioned in https://github.com/mozilla/pdf.js/pull/12663#pullrequestreview-539149098 as well since in its current form this unfortunately cannot land.

**Pull request closed without merging**